Detector/DetFCChhTrackerTkLayout/src/TkLayoutBarrel_Geo.cpp

```diff
@@ -45,13 +53,18 @@ static DD4hep::Geometry::Ref_t createTkLayoutTrackerBarrel(DD4hep::Geometry::LCD
     DD4hep::XML::Component xModulesEven = xRodEven.child("modules");
     DD4hep::XML::Component xModulePropertiesOdd = xRodOdd.child("moduleProperties");
     DD4hep::XML::Component xModulesOdd = xRodOdd.child("modules");
-    double l_overlapMargin = 0.0001;
-    DD4hep::Geometry::Tube layerShape(xLayer.rmin(), xLayer.rmax() + l_overlapMargin, dimensions.zmax());
+    DD4hep::Geometry::Tube layerShape(xLayer.rmin(), xLayer.rmax(), dimensions.zmax());
     Volume layerVolume("layer", layerShape, lcdd.material("Air"));
     layerVolume.setVisAttributes(lcdd.invisible());
     PlacedVolume placedLayerVolume = topVolume.placeVolume(layerVolume);
     placedLayerVolume.addPhysVolID("layer", layerCounter);
     DetElement lay_det(topDetElement, "layer" + std::to_string(layerCounter), layerCounter);
+    Acts::ActsExtension::Config layConfig;
+    layConfig.isLayer = true;
+    layConfig.axes = "XzY";
```

Comment:
Can you add a (few) line(s) of comment to describe what this magical string means?